src/main/scala/mylib/MyTopLevel.scala

```diff
@@ -1,67 +1,22 @@
-/*
- * SpinalHDL
- * Copyright (c) Dolu, All rights reserved.
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3.0 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library.
- */
-
 package mylib
 
 import spinal.core._
-import spinal.lib._
-
-import scala.util.Random
 
-//Hardware definition
+// Hardware definition
 class MyTopLevel extends Component {
   val io = new Bundle {
-    val cond0 = in  Bool()
-    val cond1 = in  Bool()
-    val flag  = out Bool()
-    val state = out UInt(8 bits)
+    val cond0 = in Bool ()
+    val cond1 = in Bool ()
+    val flag = out Bool ()
+    val state = out UInt (8 bits)
   }
```

Comment:
Hmm I thought we had added options for that in `.scalafmt.conf`. Maybe it is because I first formatted without configuration. I'll check if I re-do it by hand with the configuration file, to see if `scalafmt` erases it.

(Actually I must admit I prefer it this way… I never want to do any kind of code formatting by hand, so boring, and I like the one-space, unaligned, version too)

Reply:
Ok so no it does not keep the double spaces.

I restore it.